src/Evolution/Systems/GeneralizedHarmonic/DampedHarmonicGaugeQuantities.hpp

```diff
@@ -0,0 +1,354 @@
+ * by Deppe+ (2018).
+ *
+ * The covariant form of the source function \f$H_a\f$ is written as:
+ *
+ * \f{align*}
+ * H_a :=  A_{HI} [1 - R_{HI}(t)] H_a^\mathrm{init} +
+ *  [\mu_{L1} log(\sqrt{g}/N) + \mu_{L2} log(1/N)] t_a - \mu_S g_{ai} N^i / N
+ * \f}
+ *
+ * where \f$N, N^k\f$ are the lapse and shift respectively,
+ * \f$t_a\f$ is the unit normal one-form to the spatial slice,
+ *
+ * \f[
+ *  \mu_{L1} = A_{L1} R_{L1}(t) W(x^i) log(\sqrt{g}/N)^{e_{L1}}, \\
+ *  \mu_{L2} = A_{L2} R_{L2}(t) W(x^i) log(1/N)^{e_{L2}}, \\
+ *  \mu_{S} = A_{S} R_{S}(t) W(x^i) log(\sqrt{g}/N)^{e_{S}},
+ * \f]
+ *
+ * with the roll-on functions \f$ R_X(t)\f$ (with label X):
+ * \f[
+ * R_X(t) = 1 - exp[-((t - t0_X)/ \sigma_t^X)^4])\f$,
+ * \f]
+ * and the spatial roll-off function:
+ * \f[ W(x^i) = exp[-\\sigma_x (r/r_{\\mathrm{max}})^2]. \f]
+ *
+ * Note:
+ * 1) Amplitude factors \f$ A_{X} \f$ are specified as amp\_coef\_X
+ * 2) \f$ R_{X} \f$ is specified by \f$ {t0_X, \sigma_t^X}\f$, which are
+ *    taken as input as {t\_start\_X, sigma\_t\_X}.
+ * 3) Exponents \f$ e_X\f$ are specified as exp\_X.
+ * 4) The spatial weight function is specified by {r\_max, r\_width}.
+ */
+template <size_t SpatialDim, typename Frame>
+void compute_damped_harmonic_h(
+    gsl::not_null<typename GaugeH<SpatialDim, Frame>::type*> gauge_H,
+    const typename InitialGaugeH<SpatialDim, Frame>::type& GaugeH_init,
+    const Scalar<DataVector>& lapse,
+    const tnsr::I<DataVector, SpatialDim, Frame>& shift,
+    const Scalar<DataVector>& sqrt_det_spatial_metric,
+    const tnsr::aa<DataVector, SpatialDim, Frame>& spacetime_metric,
+    const double& time, const tnsr::I<DataVector, SpatialDim, Frame>& coords,
+    // Scaling coeffs in front of each term
+    const double& amp_coef_HI,
+    const double& amp_coef_L1,
+    const double& amp_coef_L2,
+    const double& amp_coef_S,
+    // exponents
+    const int& exp_L1, const int& exp_L2, const int& exp_S,
+    // roll on function parameters for lapse / shift terms
+    const double& t_start_HI, const double& sigma_t_HI,
+    const double& t_start_L1,  const double& sigma_t_L1,
+    const double& t_start_L2,  const double& sigma_t_L2,
+    const double& t_start_S, const double& sigma_t_S,
+    // weight function
+    const double& r_max, const double& r_width) noexcept;
+
+template <size_t SpatialDim, typename Frame>
+struct DampedHarmonicHCompute : db::ComputeTag {
+  using argument_tags = tmpl::list<InitialGaugeH<SpatialDim, Frame>,
+      ::gr::Tags::Lapse<DataVector>,
+      ::gr::Tags::Shift<SpatialDim, Frame, DataVector>,
+      ::gr::Tags::SqrtDetSpatialMetric<DataVector>,
+      ::gr::Tags::SpacetimeMetric<SpatialDim, Frame, DataVector>,
+      ::Tags::Time, GaugeHRollOnStartTime, GaugeHRollOnTimeWindow,
+      ::Tags::Coordinates<SpatialDim, Frame>, GaugeHSpatialWeightRMax<Frame>,
+      GaugeHSpatialWeightDecayWidth<Frame>>;
+
+  using volume_tags = tmpl::list<>;
```

Comment:
This type alias is used by the `InterfaceComputeItem` in Domain/Tags.hpp. It determines which `argument_tags` are taken from the databox directly, as opposed to taking them as `Tags::Interface<directions,argument_tag>`. You need this type alias if you want the compute item to be evaluated at the interface only.

For a compute item like this you probably won't need this functionality, so just remove this line.